vignettes/TET2.Rmd

```diff
@@ -15,10 +15,8 @@ vignette: >
 knitr::opts_chunk$set(echo = TRUE)
 knitr::opts_chunk$set(collapse = TRUE, comment = "#>")
 library(devtools)
-load_all("./") 
-# the load_all("./") is necessary to make the code run (DNAme does not work without it)
```

Comment:
Right! But from my experience load_all("./") doesn't work without downloading packages limma and GEOquery! Did you install these packages before or after running load_all("./")?